apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -165,6 +224,12 @@ const FinishStudentAccount: React.FunctionComponent<{
             iconStyle: 'solid',
             title: 'arrow-right',
           }}
+          disabled={
+            name === '' ||
+            age === '' ||
+            state === '' ||
+            (isParent && parentEmail === '')
+          }
```

Comment:
LOVE this! Will really cut down on the save button being pressed 🤩 

It seems like the only way to see the error messages is to set the fields and unset them, then, right?

Reply:
Yep!